sklearn/svm/_base.py

```diff
@@ -682,10 +686,26 @@ def n_support_(self):
         if svm_type in (0, 1):
             return self._n_support
         else:
-            # SVR and OneClass
+            # SVR, NuSVR, and OneClass
```

Comment:
I think we need to have the subclasses define the `n_support_` property so it does not show up in the documentation when we remove `n_support_`.

In the other words, `SVR`, `NuSVR` and `OneClass` has a deprecated property and `BaseSVC` defines `n_support_`. Then in 1.3 we can remove the property in `SVR`, `NuSVC` and `OneClass`.